release_notes/ocp_3_3_release_notes.adoc

```diff
@@ -173,20 +172,24 @@ secondary source repository.
 
 *Support for the New schema2 Storage Format for Images*
 
-The image manifest version 2, schema2, allows multi-architecture images via a
-manifest list which references image manifests for one or more platform-specific
-versions of an image (e.g., `amd64` versus `ppc64le`). Schema 2 also supports
-the ability to hash an image's configuration, to create an ID for the image and
-provide docker content-addressable information about the image.
+The image manifest version 2, schema2 introduces the ability to hash an image's
+configuration and thus reduce manifest size, to create an ID for the image
+and provide docker content-addressable information about the image.
 
-To preserve compatibility with older docker versions, support for schema 2 must
-be manually enabled:
+Support for consuming schema 2 from external resources is enabled implicitly.
+However, images pushed to the internal registry will be converted to schema 1
+for compatibility with older docker versions. For clusters not in need of
+compatibility preservation, accepting of schema 2 during pushes can be
+manually enabled:
 
 ----
 $ oc login -u system:admin
 $ oc set env dc/docker-registry -n default REGISTRY_MIDDLEWARE_REPOSITORY_OPENSHIFT_ACCEPTSCHEMA2=true
 ----
 
+Manifest lists, providing support for Multi-architecture images, introduced by
```

Comment:
"Manifest lists, introduced by schema 2 to provide support for multi-architecture images (e.g., `amd64` versus `ppc64le`), are not yet supported in {product-title} 3.3."